woocommerce/woocommerce
# Update color palettes when AI is no available

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:




This PR introduces a list of default color palettes that will be used when the AI is not available in the assembler, 

Closes https://github.com/woocommerce/woocommerce/issues/41619
Depends on https://github.com/woocommerce/woocommerce/pull/41656


### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:
Set up woocommerce-gutenberg-products-block from https://github.com/woocommerce/woocommerce-blocks/pull/11890

1. Create a new site either via local environment or JN. Make sure to install WooCommerce Beta Tester as well.
2. Enable `customize-store` feature flag in `Tools -> WCA Test Helper -> Features`
3. Go to `WooCommerce -> Home` and click on `Customize your store` task.
4. Open your browser's inspector and use Chrome's content override [feature](https://developer.chrome.com/docs/devtools/overrides/) to change the content of https://status.openai.com/api/v2/status.json call. Change `status.indicator` to `major`. 
5. Refresh the page, go through the process.
6. Click on `Change the color palette` and check the palettes are the ones expected on https://github.com/woocommerce/woocommerce/issues/41619 and the copy is `Choose the color palette that best suits your brand. Want to change it? Create your custom color palette below, or update it later in Editor | Styles.`

7. Repeat the process removing the overridden response and make sure the color palettes are not the default and the copy has changed to `Based on the info you shared, our AI tool recommends using this color palette. Want to change it? You can select or add new colors below, or update them later in Editor | Styles.`



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

Display more color palettes when AI is not available and update the copy.
#### Comment 

</details>
